docs/guide/interactions/context_menus.rst

```diff
@@ -0,0 +1,47 @@
+:orphan:
+
+.. currentmodule:: discord
+
+.. _guide_interactions_context-menus:
+
+Context Menus Guide
+====================
+
+Context menus allow for commands to be triggered through a context menu upon right clicking a related object in the Discord UI,
+then selecting the name of the command to run from the ``Apps`` menu.
+
+
+Basic Usage
```

Comment:
It's worth mentioning that context menus can *technically* be used in cogs, but not in the traditional sense. You could add a note on how it can be used within cogs and just linking it back to this issue: https://github.com/Rapptz/discord.py/issues/7823#issuecomment-1086830458

Reply:
Linking to github issues isn't ideal for a guide, all the information should ideally be contained within. I'd say that it will suffice to say that context menus aren't designed to put in cogs, and if you want to you'll need to handle creating the ContextMenu yourself. Optionally you could include a code example in the note.

Reply:
Yea that makes sense. I wasn't really sure if it needs to be included or not

Reply:
Can you guys take a look at the updated version of this and let me know what you think? I am not sure it is worth creating a code example for as it expands beyond the typical/expected usage of a context menu.

Reply:
That looks good, it probably doesn't need more than that.